internal/task/task/bs/format_stop.go

```diff
@@ -0,0 +1,218 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveAdm
+ * Created Date: 2022-11-16
+ * Author: guiming liang (demoliang)
+ */
+
+package bs
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/opencurve/curveadm/cli/cli"
+	"github.com/opencurve/curveadm/internal/configure"
+	os "github.com/opencurve/curveadm/internal/configure/os"
+	"github.com/opencurve/curveadm/internal/task/context"
+	"github.com/opencurve/curveadm/internal/task/step"
+	"github.com/opencurve/curveadm/internal/task/task"
+	"github.com/opencurve/curveadm/internal/utils"
+)
+
+type (
+	deleteFSTab struct {
+		host       string
+		device     string
+		mountPoint string
+		oldUUID    *string
+		uuid       string
+		curveadm   *cli.CurveAdm
+	}
+)
+
+// skipStopFormat 跳过停止format操作，当前不存在format的容器
+func skipStopFormat(containerId *string) step.LambdaType {
+	return func(ctx *context.Context) error {
+		if len(*containerId) < 0 {
+			return task.ERR_SKIP_TASK
+		}
+		return nil
+	}
+}
+
+// expression 生成要删除或者添加的fstab的sed表达式
+func (s *deleteFSTab) expression(express2del, express2add *string) step.LambdaType {
+	return func(ctx *context.Context) error {
+		*express2del = fmt.Sprintf("/UUID=%s/d", *s.oldUUID)
+		*express2add = fmt.Sprintf("$aUUID=%s  %s  ext4  rw,errors=remount-ro  0  0  # %s",
+			s.uuid, s.mountPoint, WARNING_EDIT)
+		return nil
+	}
+}
+
+// execute 执行删除fstab记录
+func (s *deleteFSTab) execute(ctx *context.Context) error {
```

Comment:
The step `deleteFSTab` is similar to step `step2EditFSTab`, you can union them into one, and provide one parameter to determine whether to add record into fatab.

Reply:
resolve this issue ,remove deleteFsTabe 
add skipAdd for edit fstab to skip add record for delete